alacritty/src/config/font.rs

```diff
@@ -17,62 +14,41 @@ use crate::config::ui_config::Delta;
 /// field in this struct. It might be nice in the future to have defaults for
 /// each value independently. Alternatively, maybe erroring when the user
 /// doesn't provide complete config is Ok.
-#[serde(default)]
-#[derive(Debug, Deserialize, Clone, PartialEq, Eq)]
+#[derive(ConfigDeserialize, Default, Debug, Clone, PartialEq, Eq)]
 pub struct Font {
+    /// Extra spacing per character.
+    pub offset: Delta<i8>,
+
+    /// Glyph offset within character cell.
+    pub glyph_offset: Delta<i8>,
+
+    pub use_thin_strokes: bool,
```

Comment:
Because it's pointless. We have a method just for non-macos to always return false anyways and we have it documented in our config either way. If we wanted to warn on missing fields we wouldn't want to warn on this. So why have **more** code to remove it on a platform when there's no benefit to it at all?